scripts/upgrade_kernel.sh

```diff
@@ -4,6 +4,11 @@ set -o pipefail
 set -o nounset
 set -o errexit
 
+# Bailout if the kernel is already 6 or above (al2023)
+if [[ $(uname -r | cut -d'.' -f1) -ge 6 ]]; then
+  exit 0
+fi
+
```

Comment:
Instead of changing this script, can we just delete the template provisioner that uses it, during `transform-al2-to-al2023-config`?

Reply:
done